packages/kbn-interpreter/.gitignore

```diff
@@ -0,0 +1,4 @@
+/common
```

Comment:
Did this file get removed?  I tried looking for it in master and can't find it, but when I look at the file history, I don't see it being deleted.  Maybe file history doesn't include the delete.